.github/PULL_REQUEST_TEMPLATE/pull_request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Pull request
+about: Thanks for contributing!
+title: ''
+labels:
+assignees: ''
+
+---
+
+# Summarize your changes
+Example: 
+* Updated code to follow new syntax and bumped forc version in CI
+* Fixed a logical flaw in the minting function that allowed a user to mint more than once
+
+Select any that apply
+- [ ] Bug fix
+- [ ] New feature
+- [ ] Documentation
+- [ ] Quality of life (refactoring, restructoring repository, deprecating old feature...)
+- [ ] Other (describe below)
```

Comment:
Again, labels should be used instead of text inside the PR. Now, external contributors can't apply labels, and it doesn't look like GitHub supports PR forms.

One solution is to instead have a CI workflow that checks to make sure a PR has at least one label from a set, e.g. https://github.com/celestiaorg/celestia-node/blob/55b7b7e33c4d6520fa01361d93b8d2dcd1117c15/.github/workflows/labels.yml#L14